package.json

```diff
@@ -7,20 +7,46 @@
     "url": "https://github.com/withastro/astro.git"
   },
   "scripts": {
-    "release": "yarn build:all && changeset publish",
-    "benchmark": "yarn workspace astro run benchmark",
-    "build": "yarn build:core",
-    "build:one": "lerna run build --scope",
-    "build:all": "lerna run build --scope \"{astro,create-astro,@astrojs/*}\"",
-    "build:core": "lerna run build --scope \"{astro,@astrojs/parser,@astrojs/markdown-remark}\"",
-    "dev": "yarn dev:core --parallel --stream",
-    "dev:one": "lerna run dev --scope --parallel --stream",
-    "dev:all": "lerna run dev --scope \"{astro,create-astro,@astrojs/*}\" --parallel --stream",
-    "dev:core": "lerna run dev --scope \"{astro,@astrojs/parser,@astrojs/markdown-remark}\" --parallel --stream",
-    "format": "prettier -w .",
+    "release": "yarn build && changeset publish",
+    "build": "turbo run build --no-deps --scope=astro --scope=create-astro --scope=\"@astrojs/*\"",
+    "dev": "turbo run dev --no-deps --no-cache --parallel --scope=astro --scope=create-astro --scope=\"@astrojs/*\"",
```

Comment:
Dropped the `build:one|all|core` as well as `dev:one|all|core`. 
`yarn build` and `yarn dev` should be super speedy now!